ci/tests/sonarqubetest/features/sonarqube_auth.feature

```diff
@@ -0,0 +1,28 @@
+Feature: sonarqube authentication and authorisation
+  In order to have a valid sonarqube authentication
+  As a sonarqube user
+  I am able to view the assigned project
+
+  @unit
+  Scenario: Can access login screen
+    Given there is a sonarqube install
+    When I access the login screen
+    Then sonarqube should be unlocked
```

Comment:
Not sure whether this has much value, as Sonar doesn't have the same kind of locking as Jenkins (which I think means the initial password screen has been cleared). It looks like just pinging the login URL.

Reply:
That Scenario was present from the previous sonar test https://github.com/FINkit/sonarqube-boshrelease/blob/master/ci/tests/sonarqubetest/godogs_test.go. Think you are right we will remove the test as suggested